ietf-optical-impairment-topology.tree

```diff
@@ -14,7 +14,7 @@ module: ietf-optical-impairment-topology
        +--ro media-channel-group* [i]
        |  +--ro i                 int16
        |  +--ro media-channels* [flexi-n]
-       |     +--ro flexi-n      uint16
+       |     +--ro flexi-n      int16
```

Comment:
The proposed change is correct and aligned with the latest version of layer0-types. The value of n in G.694.1 is defined as "a positive or negative integer including 0" so it should be an int16 and not an uint16